src/passes/function_passes/CheckPointPass.cpp

```diff
@@ -163,6 +172,11 @@ struct CheckPointPass: public FunctionPass {
               }
               ReplaceInstWithInst(&oldCallInst, callInst);
             } else {
+              // The function is not an __unopt_ function. Insert a  stackmap
+              // call after the call instruction at `it` to record the return
```

Comment:
`it`?

Reply:
The current iterator. I'll make the comment clearer.